PusherServer/BatchTriggerBody.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace PusherServer
+{
+    /// <summary>
+    /// Represents the payload to be sent when triggering events
+    /// </summary>
+    class BatchTriggerBody
+    {
+        public BatchEvent[] batch { get; set; }
```

Comment:
It has to be `batch`, it's the key we desire in the serialised output. Unless there are "annotations" we can add to tell the serialiser otherwise.